src/button/button.tsx

```diff
@@ -28,7 +30,11 @@ const DefaultComponent = (props: ButtonProps, ref: ButtonRef) => {
         <Main
             ref={ref}
             data-testid={otherProps["data-testid"] || "button"}
-            disabled={disabled}
+            disabled={disabled && !focusableWhenDisabled}
+            aria-disabled={disabled}
+            aria-busy={loading}
+            tabIndex={disabled && focusableWhenDisabled ? 0 : undefined}
```

Comment:
we can remove `tabIndex` too, since non-disabled buttons are always focusable and disabled buttons are not by default